payjoin/src/send/mod.rs

```diff
@@ -515,4 +525,19 @@ pub(crate) mod test {
                 .unwrap();
         assert_eq!(url, Url::parse("http://localhost?v=2").unwrap());
     }
+
+    #[test]
+    #[cfg(feature = "multi-party")]
+    fn test_optimistic_merge_query_param() {
+        let mut url =
+            serialize_url(Url::parse("http://localhost").unwrap(), false, None, FeeRate::ZERO, "2")
+                .unwrap();
+        append_optimisitic_merge_query_param(&mut url);
+        assert_eq!(url, Url::parse("http://localhost?v=2&optimisticmerge=true").unwrap());
+
+        let url =
+            serialize_url(Url::parse("http://localhost").unwrap(), false, None, FeeRate::ZERO, "2")
+                .unwrap();
+        assert_eq!(url, Url::parse("http://localhost?v=2").unwrap());
+    }
```

Comment:
Looks like this test and the function it tests should be in payjoin/src/send/multiparty mod

Reply:
done and done